Lib/test/certdata/make_ssl_certs.py

```diff
@@ -1,14 +1,18 @@
 """Make the custom certificate and private key files used by test_ssl
 and friends."""
 
+import argparse
 import os
 import pprint
 import shutil
 import tempfile
 from subprocess import *
 
 startdate = "20180829142316Z"
-enddate = "20371028142316Z"
+enddate_default = "20371028142316Z"
+days_default = "7000"
+
+cmdlineargs = None
```

Comment:
Making `cmdlineargs` global means the functions can't work on their own.
Could you pass it (or its parts) into the functions that need it, as an argument?